modules/installing-mirroring-disconnected-v2.adoc

```diff
@@ -0,0 +1,155 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="installing-mirroring-disconnected-v2"]
+= Mirroring images for a disconnected installation using the oc-mirror plugin V2
+include::_attributes/common-attributes.adoc[]
```

Comment:
@subhtk I am not sure what would be the right way to add documentation for v2 but if all the theortical content is same for v1 and v2, can we update examples section alone to reflect how to perform mirroring ? i.e add command how to do  disk2mirror for v2 under the disk2mirror of v1

Reply:
Regarding this, we did discuss, if it would be better to add it in V1, but with the new functionalities it might be confusing a bit. And since V1 will no longer be used later on, we thought to create to different sections for V1 and V2. Regarding the duplication, since some steps might vary for v2, it would be ideal to create a different section. let me wdyt.

Reply:
@subhtk - Yes lets leave the v2 section in, the thinking is to have some sort of warning in v1 that this will eventually be deprecated once v2 goes GA